doc/chapters/configuration/index.rst

```diff
@@ -73,40 +73,46 @@ Command line options
 ====================
 It is possible to configure SoftwareContainer with command line options.
 
-**-c or --config <path>** Path to SoftwareContainer configuration file, defaults to |sc-config-file-code|
+**-c or --config <path>** Path to SoftwareContainer configuration file,
+defaults to |sc-config-file-code|
 
-**-u or --user <uid>** Default user id to be used when starting processes in the container, defaults to ``0``.
-This option is deprecated.
+**-u or --user <uid>** Default user id to be used when starting processes in the container,
+defaults to ``0``. This option is deprecated.
 
 **-t or --timeout <seconds>** Timeout in seconds to wait for containers to shutdown,
 defaults to |shutdown-timeout-code|
 
 **-m or --manifest-dir <path>** Path to a file or directory where service manifest(s) exist,
 defaults to |service-manifest-dir-code|
 
-**-d or --default-manifest-dir <path>** Path to a file or directory where default service manifest(s) exist,
-defaults to |default-service-manifest-dir-code|
+**-d or --default-manifest-dir <path>** Path to a file or directory where default
+service manifest(s) exist, defaults to |default-service-manifest-dir-code|
 
-**-b or --session-bus** To use the session bus on D-Bus, instead of the system bus. Defaults to |use-session-bus-code|
+**-b or --session-bus** To use the session bus on D-Bus, instead of the system bus.
+Defaults to |use-session-bus-code|
 
 **-h or --help** Display usage and exit
 
 
 LXC config
 ==========
 
-LXC is configured partly by the LXC template ``libsoftwarecontainer/lxc-softwarecontainer.in`` which is configured
-and installed by CMake. This is passed to LXC by SC.
+LXC is configured partly by the LXC template ``libsoftwarecontainer/lxc-softwarecontainer.in`` which
+is configured and installed by CMake. This is passed to LXC by SC.
 
-The other source of LXC specific configuration is in ``libsoftwarecontainer/softwarecontainer.conf.in`` which is
-configured and installed by CMake to |lxc-config-path-code|.
+The other source of LXC specific configuration is in ``libsoftwarecontainer/softwarecontainer.conf.in``
+which is configured and installed by CMake to |lxc-config-path-code|.
+
+LXC has other options which can effect how SoftwareContainer behaves. One such example is the
+default path to use for containers. This option can be configured by setting ``lxc.lxcpath`` in
+the LXC configuration file (typically found in ``/etc/lxc/lxc.conf`` or ``/usr/etc/lxc/lxc.conf``).
```

Comment:
I would prefer to have this as a bullet list instead. It is easier to read when scrolling thorough the text and has better support for more instances.